client/fresheyes-bot/src/util.ts

```diff
@@ -65,6 +65,19 @@ export function getIssueBody<T extends Record<string, any>>(arg: T) {
   return { body };
 }
 
+export function generateIssueBody<T extends Array<Record<string, any>>>(arg: T) {
+  const authors = Array.from(new Set(arg.map((item) => item?.user?.login))).length;
+  const comments = arg.length;
+
+  return [
+    {
+      body: `There were ${comments} comments left by ${authors} reviewers for the pull request`,
+      created_at: arg[0].created_at,
```

Comment:
Are we using just the 1st date in the comment array? If so, what about the other dates for different comments?

Reply:
I decided to use the first date because we are aggregating all the comments and returning one object. There is no logical I can use to decide which date to be selected or used.
We added the created_at property so we can add comments chronologically when recreating them, but since we're aggregating all issue comments there is no specific need for that for issue comments, unless we want to add issue comments first before adding reviews and pull reviews.

Reply:
What do you think @Extheoisah

Reply:
Fair point. We can add the date range to the comment perhaps. But lets skip that for now. It can come in another pr as an enhancement after we've thought about it a lil bit further

Reply:
Great